content/docs/styled-system/color-mode.mdx

```diff
@@ -10,8 +10,23 @@ Chakra UI comes with built-in support for managing color mode in your apps.
 By default, most of Chakra's components are dark mode compatible. In some
 scenarios, you might need to make your component respond to color mode.
 
-> **Tip:** Chakra stores the color mode in `localStorage` and appends a
-> className to the `body` to ensure the color mode is persistent.
+> **Tip:** Chakra stores the color mode in `localStorage` or in `cookies` and appends a
+> className to the `body` to ensure the color mode is persistent.  
+> In case you need to reset the color mode, you must delete the item from `localStorage` or `cookies`, so on next page load the value is initialized like the first time user visited the page.
+
+## Introduction to Color Mode
+
+In the recent years, operative systems (`system` for this guide) give user the option to choose from Light or Dark color mode.
```

Comment:
Can we change this to `operating systems`?

Reply:
Sure, to be consistent i replace every `operative system` occurrence with `operating system`

Reply:
Included in commit 1d1ec8bb6a4a658357504dbde6b8077c8ac6cfb0